_quarto.yml

```diff
@@ -17,14 +17,17 @@ website:
     search: true
     right:
       - about.qmd
-      - rladies.qmd
       - sponsors.qmd
-      - resources.qmd
       - slides.qmd
+      - rladies.qmd
       - codeofconduct.qmd
       - icon: github
         href: https://github.com/SheffieldR
-      - icon: twitter
+      - icon: calendar-event
+        href: https://www.meetup.com/sheffieldr-sheffield-r-users-group/
+      - icon: slack
+        href: https://sheffieldr.slack.com
+      - icon: twitter-x
         href: https://twitter.com/Sheffield_R_
       - icon: rss
         href: index.xml
```

Comment:
I'd be inclined to change the order of RSS and mastodon here so that RSS appears last before the search icon.